CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ scheme are considered to be bugs.
 
 ### Miscellaneous
 
+* [#981](https://github.com/hashie/hashie/pull/981): Stop shipping tests with the gem - [@michaelherold](https://github.com/michaelherold).
```

Comment:
If I am being pedantic, "Exclude tests from gem release" may sound less like "stop! don't do this" :)